build/template-publish-packages-and-symbols.yaml

```diff
@@ -35,3 +35,10 @@ steps:
   inputs:
     SearchPattern: '${{ parameters.SymbolPublishWildcard }}'
     SymbolServerType: TeamServices
+
+- task: PublishSymbols@2
+  displayName: 'Publish PDBs'
+  inputs:
+    SearchPattern: '**/bin/**/*.pdb'
```

Comment:
can we add this to the parameters?

Reply:
Yes, Gladwin is right. The search pattern here is too broad, and it will include tons of PDB files such as the ones that come from testing DLLs. The search pattern should be similar to  '**/bin/**/microsoft.identity.client.*'

Reply:
we have a similar search pattern on line 7
`SymbolPublishWildcard: '**/bin/**/microsoft.identity.client.*'`

Reply:
Travis is right about having the pattern. But it does not publish pdbs. I do not know the internals; but it looks like they are skipped by default. If they would publish everything qualified by "microsoft.identity.client.*", there will be too many files getting published. But again that is my hypothesis and I have not looked at the code.

Reply:
Currently (for the current pattern) in the publish symbols task logs it does say it's uploading and succeeding processing PDBs and DLLs for both MSAL and MSAL.Desktop. So maybe then it doesn't upload into the correct location or something?